source/puppet/4.10/lang_data_number.md

```diff
@@ -109,28 +122,27 @@ The full signature for `Integer` is:
 
 All of these parameters are optional. They must be listed in order; if you need to specify a later parameter, you must specify values for any prior ones.
 
-Position | Parameter        | Data Type | Default Value | Description
----------| -----------------|-----------|---------------|------------
-1 | Min value | `Integer` | negative infinity | The minimum value for the integer. This parameter accepts the special value `default`, which will use its default value.
-2 | Max value | `Integer` | infinity | The maximum value for the integer. This parameter accepts the special value `default`, which will use its default value.
+Position | Parameter        | Data type | Default value     | Description
+---------| -----------------|-----------|-------------------|-----------------------------
+1        | Minimum value    | `Integer` | negative infinity | The minimum value for the integer. This parameter accepts the special value `default`, which uses its default value.
+2        | Maximum value    | `Integer` | infinity          | The maximum value for the integer. This parameter accepts the special value `default`, which uses its default value.
 
 Practically speaking, the integer size limit is the range of a 64-bit signed integer (−9,223,372,036,854,775,808 to 9,223,372,036,854,775,807), which is the maximum size that can roundtrip safely between the components in the Puppet ecosystem.
```

Comment:
The term "size" is awkward here - since that is a positive number - the number of distinct values in the range of values a 64 bit integer can describe. How about saying "the range of integer values" instead?